index.html

```diff
@@ -229,47 +247,72 @@ <h3>Volunteer</h3>
   <div class="container-fluid">
     <div class="row no-gutters">
       <div class="col-4">
-        <img src="src/img/IMG_2580.JPG" class="img-rounded img-thumbnail" />
+        <img src="src/img/photos/IMG_2580.JPG" class="img-rounded img-thumbnail" />
       </div>
       <div class="col-4">
-        <img src="src/img/IMG_2630.JPG" class="img-rounded img-thumbnail" />
+        <img src="src/img/photos/IMG_2630.JPG" class="img-rounded img-thumbnail" />
       </div>
       <div class="col-4">
-        <img src="src/img/IMG_2852.JPG" class="img-rounded img-thumbnail" />
+        <img src="src/img/photos/IMG_2852.JPG" class="img-rounded img-thumbnail" />
       </div>
       <div class="w-100">
       </div>
     </div>
 
-    <!-- Newsletter and Link -->
-    <div class="container-fluid pt-5 pb-5 bg-black text-light" id="contact">
-      <div class="container text-center">
-        <h2 class="font-weight-bold text-uppercase text-center pb-2">
-          Never miss our event
-        </h2>
-        <a class="btn btn-primary m-2 text-light" href="https://mailchi.mp/33c3df7619af/pyconlt-2019">
-          Newsletter
-        </a>
-        <div class="row pt-5 justify-content-center">
-          <div class="col-auto">
-            <a href="https://www.facebook.com/PyConLT/"><img class="social-icon" src="src/img/facebook.svg" /></a>
-          </div>
-          <div class="col-auto">
-            <a href="https://twitter.com/PyConLT"><img class="social-icon" src="src/img/twitter.svg" /></a>
-          </div>
-          <div class="col-auto">
 
-            <a href="https://www.linkedin.com/company/PyConLT"><img class="social-icon"
-                src="src/img/linkedin.svg" /></a>
+
+    <footer class="footer font-small blue pt-4">
+      <div class="container-fluid text-center text-md-left">
+        <div class="row">
+          <div class="col-md-6 mt-md-0 mt-3">
+            <h5 class="text-uppercase">PyConLT</h5>
+            <p>PyCon Lithuania is community organized event.
+              PyConLT takes place anually. It is non-profit event.
+            </p>
+          </div>
+          <hr class="clearfix w-100 d-md-none pb-3">
+          <div class="col-md-3 mb-md-0 mb-3">
+            <h5 class="text-uppercase">Links</h5>
+            <ul class="list-unstyled">
+              <li>
+                <a href="#!">PyConLT 2020 (canceled due to covid)</a>
+              </li>
+              <li>
+                <a href="#!">PyConLT 2019</a>
+              </li>
+              <li>
+                <a href="#!">PyConLT 2018</a>
+              </li>
+              <li>
+                <a href="#!">PyConLT 2017</a>
+              </li>
+            </ul>
```

Comment:
Empty links? maybe add them then we have what to add?

Reply:
Will remove.